### PR TITLE
Update Packages.md

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Removed `InkRecognizer.UWP.Stroke` `InkRecognizer.WPF.Stroke` and `InkRecognizer` from the repo.
Excluded `SnippetGenerator` and `docgen` from docsettings.
Ran Doc-Warden to update `package.md`

### CODE REVIEW

File: packages.md [hunk 1]
```diff
@@ -3,24 +3,28 @@
...
-| [`Azure.Core`]( sdk/core/Azure.Core/src/Azure.Core.csproj )| [Readme](sdk/core/Azure.Core/README.md) |  N/A  | [![Azure.Core](https://img.shields.io/nuget/vpre/Azure.Core.svg)]( https://www.nuget.org/packages/Azure.Core/ ) |
-| [`Azure.Data.AppConfiguration`]( sdk/appconfiguration/Azure.Data.AppConfiguration/src/Azure.Data.AppConfiguration.csproj )| [Readme](sdk/appconfiguration/Azure.Data.AppConfiguration/README.md) |  N/A  | [![Azure.Data.AppConfiguration](https://img.shields.io/nuget/vpre/Azure.Data.AppConfiguration.svg)]( https://www.nuget.org/packages/Azure.Data.AppConfiguration/ ) |
-| [`Azure.Identity`]( sdk/identity/Azure.Identity/src/Azure.Identity.csproj )| [Readme](sdk/identity/Azure.Identity/README.md) |  N/A  | [![Azure.Identity](https://img.shields.io/nuget/vpre/Azure.Identity.svg)]( https://www.nuget.org/packages/Azure.Identity/ ) |
+| [`Azure.AI.InkRecognizer`]( sdk/cognitiveservices/InkRecognizer/src/Azure.AI.InkRecognizer.csproj )|  N/A  |  N/A  | [![Azure.AI.InkRecognizer](https://img.shields.io/nuget/vpre/Azure.AI.InkRecognizer.svg)]( https://www.nuget.org/packages/Azure.AI.InkRecognizer/ ) |
```
Comment (weshaggard) [inline packages.md]: Please exclude the InkRecognizer packages. In fact would you mind submitting a PR to remove those from the repo.

File: packages.md [hunk 2]
```diff
@@ -179,10 +186,12 @@
...
 | [`Microsoft.Rest.ClientRuntime.Etw`]( sdk/mgmtcommon/ClientRuntime.Etw/Microsoft.Rest.ClientRuntime.Etw.csproj )| [Readme](sdk/mgmtcommon/ClientRuntime.Etw/README.md) |  N/A  | [![Microsoft.Rest.ClientRuntime.Etw](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Etw.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Etw/ ) |
 | [`Microsoft.Rest.ClientRuntime.Log4Net`]( sdk/mgmtcommon/ClientRuntime.Log4Net/Microsoft.Rest.ClientRuntime.Log4Net.csproj )| [Readme](sdk/mgmtcommon/ClientRuntime.Log4Net/README.md) |  N/A  | [![Microsoft.Rest.ClientRuntime.Log4Net](https://img.shields.io/nuget/vpre/Microsoft.Rest.ClientRuntime.Log4Net.svg)]( https://www.nuget.org/packages/Microsoft.Rest.ClientRuntime.Log4Net/ ) |
+| [`SnippetGenerator`]( eng/SnippetGenerator/SnippetGenerator.csproj )|  N/A  |  N/A  |  N/A  |
```
Comment (weshaggard) [inline packages.md]: These 2 should be excluded.

File: packages.md (hunk 1 above)
Comment (chidozieononiwu) [reply packages.md]: Removed `InkRecognizer.UWP.Stroke`, `InkRecognizer.WPF.Stroke` and `InkRecognizer` from the repo.

File: packages.md (hunk 2 above)
Comment (chidozieononiwu) [reply packages.md]: Excluded `SnippetGenerator` and `docgen` from docsettings.

**Review by weshaggard — APPROVED** [review]

**Pull request merged**